templates/element/routes_panel.php

```diff
@@ -3,13 +3,49 @@
  * @var \Cake\Routing\Route\Route[] $routes
  * @var string $matchedRoute
  */
+
+use Cake\Core\Plugin as CorePlugin;
 use Cake\Utility\Hash;
 
 $routes = Cake\Routing\Router::routes();
+
+$amountOfRoutesPerGroup = [];
+foreach ($routes as $route) {
+    if (empty($route->defaults['plugin'])) {
+        if (!array_key_exists('app', $amountOfRoutesPerGroup)) {
+            $amountOfRoutesPerGroup['app'] = 0;
+        }
+        $amountOfRoutesPerGroup['app']++;
+    } else {
+        if (!array_key_exists($route->defaults['plugin'], $amountOfRoutesPerGroup)) {
+            $amountOfRoutesPerGroup[$route->defaults['plugin']] = 0;
+        }
+        $amountOfRoutesPerGroup[$route->defaults['plugin']]++;
+    }
+}
+
+$plugin_names = [];
+foreach (CorePlugin::loaded() as $plugin_name) {
```

Comment:
All the var names are wrong, see also below.

Reply:
god damn I am so used to that 😅

Reply:
I think I got everything now